src/adapters/IDB.js

```diff
@@ -207,49 +306,28 @@ export default class IDB extends BaseAdapter {
    * @return {Promise}
    */
   list(params={filters: {}}) {
-    // Extract from `params.filters` the fields that are indexed.
-    const filteredFields = Object.keys(params.filters);
-    const indexedFields = filteredFields.filter(f => {
-      return ["id", "_status", "last_modified"].indexOf(f) !== -1;
-    });
-    // Since indices were created on single-columns, use the first one only.
-    const indexField = indexedFields[0];
-
+    const {filters} = params;
+    const indexField = findIndexedField(filters);
+    const value = filters[indexField];
```

Comment:
Now I remember why I didn't do that; `indexField` is used in the next `.then()` callback so we can drop it from filters to be applied in-memory.